draft-ietf-spice-sd-cwt.md

```diff
@@ -581,7 +581,158 @@ The following example contains claims needed to demonstrate redaction of key-val
+The fourth disclosure unblinds the inspection region.
+
+The fifth disclosure unblinds the earliest inspection record, and the last disclosure unblinds the inspector_license_number for that record.
+
+Clients start unblinding claims for which they have blinded claim hashes. They continue descending until there are no blinded claim hashes at any level of the hierarchy for which they have a corresponding disclosure.
+
+~~~ cbor-diag
+/ sd_claims / 17 : [ / these are the disclosures /
+    <<[
+        /salt/   h'e3aa33644123fdbf819ad534653f4aaa',
+        /claim/  504,   / inspection 17-Jan-2023 /
+        /value/  [
+                     59(h'2893a00665f1ca2cfeb7456e1eeb8eba
+                          f21d5c12a73d9fbcb8902822f3ecb635'),
+                     59(h'92c0262b0ed6891c6e46d6fca5554caf
+                          79bd8d05c74dbb06a25c9edd304c6e22'),
+                     {
+                         500: True,
+                         502: 17183928,
+                         simple(59): [
+                           h'0ad0f76dcb7fd812ca64c3ada3f543be
+                              96d0e351e1e576fbab5cb659b49e599e',
+                           h'f34c3ea2292d02b92bde25e68e94acd7
+                             f1e011fd6eea6c490f841f09a7a01a48'
+                         ]
+                     }
+                 ]
+    ]>>,
+    <<[
+        /salt/   h'bae611067bb823486797da1ebbb52f83',
+        /claim/  501,   / inspector_license_number /
+        /value/  "ABCD-123456"
+    ]>>,
+    <<[
+        /salt/   h'7d2505257e7850b70295a87b3c8748e5',
+        /claim/  503,   / San Francisco location /
+        /value/  {
+                     1: "us",
+                     simple(59): [
+                       h'de03a7a0b4359511a7dc0edd8f4ebc00
+                         b5783d8a0d36e715679e23c703011d16',
+                       h'5a98ac2381cb59dee7a43daa073eab48
+                         9773e2830a0b9c4e1efd55737dbb1c06'
+                     ]
+                 }
+    ]>>,
+    <<[
+        /salt/   h'52da9de5dc61b33775f9348b991d3d78',
+        /claim/  2,   / region=California /
+        /value/  "ca"
+    ]>>,
+    <<[
+        /salt/   h'b52272341715f2a0b476e33e55ce7501',
+        /value/  {
+                     500: True,
+                     502: 1549560720,
+                     simple(59): [
+                       h'cd88763edb2485b8109613546051f606
+                         e6b822456da1bf09f604b886e1def45a',
+                       h'0a45eb75de44741bea78dc48b1898d40
+                         09601dbf567279f3042a24cee9fdcab5'
+                     ]
+                 }   / inspection 7-Feb-2019 /
+    ]>>,
+    <<[
+        /salt/   h'591eb2081b05be2dcbb6f8459cc0fe51',
+        /claim/  501,   / inspector_license_number /
+        /value/  "DCBA-101777"
+    ]>>
+]
+~~~
+
+After applying the disclosures of the nested structure above, the payload visible to the verifier would look like the following:
```

Comment:
```suggestion
After applying the disclosures of the nested structure above, the disclosed claim set visible to the verifier would look like the following:
```

Reply:
I think we want to preserve that "payload" is what is actually signed... not the transformed / validated data structure (which is the disclosed claim set)